src/commands/moderators/activeThreads.ts

```diff
@@ -97,8 +88,77 @@ export default {
   },
 } satisfies CommandHandlerConfig
 
-async function getActiveThreads(client: Client, guild: { id: string }) {
-  return (await client.rest.get(
-    Routes.guildActiveThreads(guild.id),
-  )) as RESTGetAPIGuildThreadsResult
+async function generateReport(
+  botContext: BotContext,
+  interaction: MessageComponentInteraction,
+  threads: APIThreadChannel[],
+) {
+  const tsv = [
+    [
+      'ID',
+      'Name',
+      'Channel ID',
+      'Channel Name',
+      'Last Message',
+      'Message Count',
+    ],
+    ...threads.map((thread) => {
+      const channel = thread.parent_id
+        ? botContext.client.channels.cache.get(thread.parent_id)
+        : undefined
+      return [
+        thread.id,
+        thread.name,
+        thread.parent_id ?? '-',
+        (channel && 'name' in channel && channel.name) ?? '-',
+        thread.last_message_id
+          ? toLocalDate(SnowflakeUtil.timestampFrom(thread.last_message_id))
+          : '-',
+        thread.message_count,
+      ]
```

Comment:
Can we do character escaping here, just in case?

Reply:
done, with a new utility `generateTsv` (with tests krub)